udata/templates/organization/display.html

```diff
@@ -140,7 +140,7 @@
     {% endfor %}
 
     {% if org.members %}
-    <tab id="members" header="{{ _('Members') }}">
+    <tab id="members" header="{{ ngettext('%(num)d member', '%(num)d members', org.members | length) }}">
```

Comment:
Cool, good to know! On my local instance I did not setup the workers but rabbitmq instance is up (I use `docker-compose up` to setup the app resource dependencies when I develop stuff).